browser.js

```diff
@@ -23,7 +24,7 @@ function changeTab(next) {
 }
 
 // sets interval to wait for selector to be ready before firing callback
-function waitFor(selector) {
+const waitFor = selector => {
```

Comment:
@sindresorhus is this actually required? I made the change and didn't even think if I need to or not.